controllers/controller.js

```diff
@@ -7,8 +7,18 @@ exports.getCategories = (req, res) => {
   });
 };
 
-exports.getEndpoints = (req, res) => {
+exports.getEndpoints = (req, res, next) => {
   return fs.readFile("endpoints.json", "utf-8").then((result) => {
     res.status(200).send({ endpoint: result });
   });
 };
+
+exports.getReviewId = (req, res, next) => {
+  let { review_id } = req.params;
+  fetchReviewId(review_id)
+    .then((data) => {
+      console.log(data);
+      res.status(200).send({ review: data });
```

Comment:
Yes good work with this... you are sending the data back on a key called 'review', to represent a single review, this is a good naming of your key